DockerSetup.md

```diff
@@ -0,0 +1,78 @@
+# Local DP Dash development on Docker
+
+This document outlines the set up for the development environment using docker
+
+## Requirements
+
+- [Complete all the steps up to the initialize step](./README-PNL.md#initialize)
```

Comment:
the initial steps do things like setting up singularity. Do we still need to do that? My hunch/hope is that we can greatly reduce the setup complexity with this change.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/358#issuecomment-1669999988